tokio-tcp/Cargo.toml

```diff
@@ -37,4 +37,4 @@ futures-core-preview = { version = "0.3.0-alpha.16", optional = true }
 [dev-dependencies]
 #env_logger = { version = "0.5", default-features = false }
 #net2 = "*"
-#tokio = { version = "0.2.0", path = "../tokio" }
+tokio = { version = "0.2.0", path = "../tokio" }
```

Comment:
I don't _think_ you need this change any more?

Reply:
It is needed for the test.

Reply:
Ohh, I missed the `#[tokio::test]` annotation :+1: